service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -29,40 +29,3 @@ spec:
 ----
 
 For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
-
-[id="ossm-auto-route-subdomains_{context}"]
-== Subdomains
-
-{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default. Cluster administrators can refer to the {product-title} documentation for instructions on how to enable subdomains.
-
-For example, if the following gateway is created:
```

Comment:
The example below is not about subdomains. It is a generic example showing the expected result of using IOR. So, you just need to cut the subdomain paragraph above - and perhaps rephrase this `For example...`

Reply:
You're right. Added it back, @jwendell